src/button/props.js

```diff
@@ -86,7 +86,7 @@ export const normalizeProps = memoize((props : Object, defs? : { locale? : Local
         [ BUTTON_STYLE_OPTIONS.SHAPE ]:        shape        = getButtonConfig(label, 'defaultShape'),
         [ BUTTON_STYLE_OPTIONS.BRANDING ]:     branding     = getButtonConfig(label, (layout === BUTTON_LAYOUT.VERTICAL) ? 'defaultVerticalBranding' : 'defaultBranding'),
         [ BUTTON_STYLE_OPTIONS.FUNDINGICONS ]: fundingicons = getButtonConfig(label, 'defaultFundingIcons'),
-        [ BUTTON_STYLE_OPTIONS.TAGLINE ]:      tagline      = getButtonConfig(label, 'defaultTagline'),
+        [ BUTTON_STYLE_OPTIONS.TAGLINE ]:      tagline      = style[BUTTON_STYLE_OPTIONS.TAGLINE] ? getButtonConfig(label, 'defaultTagline') : false,
```

Comment:
Why change this? Doesn't `getButtonConfig(label, 'defaultTagline')` already give the correct default value?

Reply:
It does but then if the user doesn't set the `tagline: false`, it defaults to true.

Reply:
For checkout.js do we want the tagline to default to being true? 

I was looking through the example code snippets and I think we do want the tagline on by default. This is the only code snippet I found in the v4 demo using tagline and it's explicitly setting it to false: https://developer.paypal.com/demo/checkout-v4/#/pattern/generic

Reply:
Yep defaulting to true is the expected behavior. `getButtonConfig(label, 'defaultTagline')` should return `true`.

Reply:
@gregjopa @bluepnume This was done as I was going off of the tests assuming they were the source of truth.  But no guessing that the tests were changed to match up with the incorrect code.  I've changed back and updated tests.